src/data/DateRangeSelector.ts

```diff
@@ -148,3 +149,74 @@ export function specialDateRangeToString(dateRange: SpecialDateRange): string {
       return '2022';
   }
 }
+
+export interface DateRangeRaw {
+  [key: string]: string | null;
+}
+
+const fields = ['dateSubmittedFrom', 'dateSubmittedTo', 'dateSubmitted'] as const;
```

Comment:
`fields` is not a very clear variable name within this file. These are only fields related to the submission dates but the file is more generic and about date ranges in general.